.github/workflows/test.yml

```diff
@@ -52,6 +52,19 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y libpcap-dev
+        sudo apt-get upgrade -y
+    - name: Install Cassandra CPP driver
+      if: ${{ matrix.runner == 'ubuntu-18.04' }}
+      run: |
+        echo CPP version number
+        cc --version
```

Comment:
Do we want to leave this logging stuff in the PR or is it just leftovers?

Reply:
That's reasonable. Though I doubt the `cc` version will change on the Ubuntu images we are building with.

Reply:
The cassandra driver installation needs to go into shotover-proxy/build/install_ubuntu_packages.sh otherwise the release.yml workflow will fail.

Reply:
and then removed from here.
Otherwise we would run it twice!